pyproject.toml

```diff
@@ -105,6 +106,10 @@ ignore-paths = [
 fail-on = "I"
 django-settings-module="integreat_compass.core.settings"
 
+[tool.isort]
+profile = "black"
+order_by_type = false
```

Comment:
Here as well, please don't put it in between two pylint config blocks

Reply:
When sorting them alphabetically, is it okay to put them before [tool.setuptools]? Or should they be behind the setuptools and then sorted alphabetically?

Reply:
I think both approaches are fine, although I would somehow argue that `[tool.setuptools]` is special since it is the core setting how the package is installed with pip...